parachain/primitives/router/src/outbound/tests.rs

```diff
@@ -1,11 +1,14 @@
 use frame_support::parameter_types;
 use hex_literal::hex;
-use snowbridge_core::outbound::{Fee, SendError, SendMessageFeeProvider};
+use snowbridge_core::{
+	outbound::{Fee, SendError, SendMessageFeeProvider},
+	DescribeHere,
+};
 
 use xcm::v3::prelude::SendError as XcmSendError;
 use xcm_builder::{DescribeAllTerminal, DescribeFamily, HashedDescription};
 
-pub type AgentIdOf = HashedDescription<H256, DescribeFamily<DescribeAllTerminal>>;
+pub type AgentIdOf = HashedDescription<H256, (DescribeHere, DescribeFamily<DescribeAllTerminal>)>;
```

Comment:
We're also repeating this line in the BridgeHub `xcm_config.rs` configuration, can you check?

In which case we should factor out both definitions into our [parachain/runtime/runtime-common](https://github.com/Snowfork/snowbridge/blob/main/parachain/runtime/runtime-common/src/lib.rs) crate.

Reply:
Yeah, several duplications and refactoring with https://github.com/Snowfork/snowbridge/pull/1077/commits/8c1d1d89dd5df205fbe90c4cf256252bc771494c

Btw, prefer to move to primitives as it's not only used in runtime tests.